spec/twiliosms_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'twiliosms'
+
+describe TwilioSMS do
+
+  let(:test_sid) { ENV["TWILIO_TEST_ACCOUNT_SID"] }
+  let(:test_key) { ENV["TWILIO_TEST_AUTH_TOKEN"] }
+  let(:test_mobile) { ENV["TWILIO_TEST_MOBILE"] }
+  let(:twilio) { TwilioSMS.new(test_sid, test_key) }
+
+  let(:time_plus_one) do
+    time = Time.now.strftime("%H:%M")
+    time_plus_one = Time.now.strftime("%H:%M").split(":")
+    time_plus_one[0] = time_plus_one[0].to_i
+    time_plus_one[0] += TwilioSMS::DELIVERY_TIME
+    time_plus_one[0] = time_plus_one[0].to_s
+    time_plus_one = time_plus_one.join(":")
+  end
```

Comment:
This is way too much logic to have in a test. What if that logic is wrong?
Use timecop and hardcoded strings instead.
For example, you can freeze the time using timecop to be 1pm, and then use `"Thank you! Your order was placed and will be delivered before 14:00"` in your expectation.